src/atemSwitcher.ino

```diff
@@ -151,6 +152,7 @@ void loop() {
 
     updateState();
     updateATEM();
+    muteMainAudio();
```

Comment:
@AJDurant That is a fair comment, I have renamed it.

Yeah we are now using HDMI for both cameras in S1, I think our HDMI-to-SDI converter has died but need to have a better look at it. Do I just need to update the names on the ATEM Switcher?